example/app/config.yml

```diff
@@ -65,7 +65,7 @@ default:
   # stackNameNoDash: TestCumulusIntegration
   shared_data_bucket: cumulus-data-shared
   api_distribution_url: https://example.com/
-  api_lambda_memory: 756
+  api_lambda_memory: 1024
```

Comment:
DynamoDB transactions are not supported by the version of the `aws-sdk` package supplied by AWS Lambdas.  As a result, we need to package a newer version, which increases the memory footprint of the lambda function.